mgmtsystem_nonconformity/readme/USAGE.rst

```diff
@@ -0,0 +1,45 @@
+To use this module:
+
+* Go to Management System > Nonconformities
+* Click on Create to enter the following information:
+
+* Partner : Customer, supplier or internal personnel
+* Related to: Any reference pointing to the NC (order id, project id, task id, etc.)
+* Responsible: Person responsible for the NC
+* Manager : Person managing the department or owner of the procedure
+* Filled in by: Originator of NC report
+* Origins:  The source of the NC, how was it discover
+* Procedures:  Against which procedure is the NC
+* Description: Evidence, reference to the standards
+
+* Click on Save and then on Analysis.
+
+Go to the newly created NC and fill in the following
+information in the tab named Causes and Analysis:
+
+* Causes: Add root causes
+* Analysis: Describe the results of the investigation
+* Severity: Select the severity among unfounded, minor and major
+* Immediate action: Create or select an immediate action if appropriate
+
+Click on the Save button and then on the "Action Plan" button in the top right corner.
+
+In the Actions tab, select or create new actions by entering the following
+items:
+
+* Subject: What must be done - Return to Supplier, Use As Is, Scrap, Rework,
+  Re-grade, Repair
+* Deadline: Date by which the action must be completed
+* Responsible: Person in charge for implementing the action
+* Type: Immediate, corrective or preventive actions or improvement opportunity
+* Description: Details of the action
+
+When the action is created, a notification is sent to the person responsible
+for the action.
+
+Enter comments into the input field below the "Plan Review" section, those comments are required to reach the next stage.
+
+To begin the work on the planned Actions change the stage of the NC to open by clicking on the "In Progress" button in the top right corner.
+
+When all actions of the plan are done, their effectiveness must be evaluated
+before closing the NC.
```

Comment:
For me better:
`before closing the NC clicking on the "Closed" on stage sequence in the top right corner.`